.github/workflows/docker_build.yml

```
@@ -1,9 +1,9 @@
 name: "Docker build"
 on:
   push:
-    branches: [main]
+    branches: [ rails-api ]
```

```suggestion
    branches: [ main, rails-api ]
```
I think we should include both for the time being. It shouldn't run on main branch just yet because the actions only really run from whatever is declared in main branch